src/yb/master/master-path-handlers.cc

```diff
@@ -1462,6 +1483,11 @@ Status MasterPathHandlers::Register(Webserver* server) {
       "/cluster-config", "Cluster Config",
       std::bind(&MasterPathHandlers::CallIfLeaderOrPrintRedirect, this, _1, _2, cb), is_styled,
       false);
+  cb = std::bind(&MasterPathHandlers::HandleGetClusterConfigJSON, this, _1, _2);
+  server->RegisterPathHandler(
+      "/api/v1/cluster-config", "Cluster Config JSON",
+      std::bind(&MasterPathHandlers::CallIfLeaderOrPrintRedirect, this, _1, _2, cb), is_styled,
```

Comment:
The is_styled value here causes output to be surrounded by HTML, you will need to pass false. Please follow the example for HandleHealthCheck.

Reply:
Done